docs/classes/_ricardiancontractfactory_.ricardiancontractfactory.md

```diff
@@ -31,7 +31,7 @@
 
```

Comment:
Maybe in future PRs separating out the `docs` updates into another PR would be nice, it made it a bit harder to find the significant code changes

Reply:
Yeah, I'm torn between keeping the PR unclutter vs making sure the docs are properly in-sync with the code it documents in the same PR.

Maybe an alternative would be to create a PR without the docs updated. Then after the PR is ready to go, generate and push one more commit to it with JUST the docs. Since they're generated they don't really need review. But it keep the code and corresponding docs tied closely together. Thoughts?

Reply:
I actually don't mind having a follow up separate PR with the doc changes, quickly after the code changes are merged into develop, just so that can also have a quick review. I do get your point with having them pushed together though, but since for this repo its mostly auto generated docs, I don't think it's as a big of a deal, since technically you could just generate them from the checked in code. If there were manually generated docs with important new information / breaking changes etc, I think then i lean more towards wanting the documentation more tightly coupled with the code